app/models/user.rb

```diff
@@ -1,2 +1,14 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] As indicated from the ERD design, just like `posts`, I think including a `foreign_key` of `:author_id` on `comments` and `likes` is more correct as far as the project requirement is concerned.

Reply:
Thank you for the correction. I have implemented the changes.